cmd/runtimetest/main.go

```diff
@@ -244,6 +245,45 @@ func validateROPaths(spec *rspec.Spec) error {
 	return nil
 }
 
+func getThisCgroupPath(subsystem string) (string, error) {
+	f, err := os.Open("/proc/self/cgroup")
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+
+	s := bufio.NewScanner(f)
+	for s.Scan() {
+		if err := s.Err(); err != nil {
+			return "", err
+		}
+
+		text := s.Text()
+		parts := strings.Split(text, ":")
+
+		for _, subs := range strings.Split(parts[1], ",") {
+			if subs == subsystem {
+				return parts[2], nil
+			}
+		}
+	}
+	return "", fmt.Errorf("cgroup %v not found", subsystem)
+}
+
+func validateCgroupsPath(spec *rspec.Spec) error {
+	fmt.Println("validating cgroupsPath")
+	expectedPath := spec.Linux.CgroupsPath
+	actualPath, err := getThisCgroupPath("devices")
```

Comment:
Thanks for your information. I misunderstood the referred link.
I will make more investigation and update the PR later.